scripts/Partitions.base.swift

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Code is auto-generated. DO NOT EDIT!
+
+public let partitions = #"""
```

Comment:
This file is prepended to the partitions JSON to create a Swift string declaration at `Sources/Core/AWSSDKPartitions/Sources/AWSSDKPartitions/Partitions.swift`.
`"""#` is appended after the JSON to close the string.

(Creation of this file is performed in the build system.)